pandas/tests/series/test_analytics.py

```diff
@@ -336,6 +336,31 @@ def test_describe(self):
                           index=['count', 'unique', 'top', 'freq'])
         tm.assert_series_equal(result, expected)
 
+        start = Timestamp(2018, 1, 1)
```

Comment:
can u add a new test and/or parametrize on the tz

Reply:
done